services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1119,4 +1121,49 @@ module.exports = {
       });
     },
   },
+
+  /**
+   *
+   */
+  Mutation: {
+    updateContentCompany: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
+      const type = 'platform/content/company';
+      const { id, ...payload } = input;
+      const keys = Object.keys(payload);
+      const body = new Base4RestPayload({ type });
+      keys.forEach(k => body.set(k, payload[k]));
+      body.set('id', id);
+      await base4rest.updateOne({ model: type, id, body });
+      const { fieldNodes, schema, fragments } = info;
```

Comment:
Consolidate this projection handling... should ultimately defer to `findOne` directive.